src/main/java/com/devcycle/sdk/server/local/api/DVCLocalClient.java

```diff
@@ -239,4 +306,71 @@ private void validateUser(User user) {
   private boolean isValidServerKey(String serverKey) {
     return serverKey.startsWith("server") || serverKey.startsWith("dvc_server");
   }
+
+  private NullableString createNullableString(String value)
+  {
+    return value == null ? NullableString.newBuilder().setIsNull(true).build() : NullableString.newBuilder().setIsNull(false).setValue(value).build();
```

Comment:
A bit of formatting helps make this more readable:
```suggestion
    return value == null 
        ? NullableString.newBuilder().setIsNull(true).build() 
        : NullableString.newBuilder().setIsNull(false).setValue(value).build();
```